aws/cloudformation/iam.yml.erb

```diff
@@ -214,31 +214,6 @@ Resources:
         - Key: sqlworkbench-team
           Value: codedotorg
 
-# TODO: Migrate to REDTeam_Analyst
-  DataAnalystRole:
-    Type: AWS::IAM::Role
-    Properties:
-      RoleName: DataAnalyst
-      Path: /dataanalyst/
-      AssumeRolePolicyDocument:
-        Version: 2012-10-17
-        Statement:
-          - Effect: Allow
-            Principal: {Federated: !Sub "arn:aws:iam::${AWS::AccountId}:saml-provider/Google"}
-            Action: sts:AssumeRoleWithSAML
-            Condition:
-              StringEquals:
-                SAML:aud: https://signin.aws.amazon.com/saml
-      ManagedPolicyArns:
-        - !Ref DataAnalystPolicy
-        - !Ref DataAnalystDmsReadOnlyPolicy
-        - !Ref DataAnalystRedshiftPolicy
-        - arn:aws:iam::aws:policy/AmazonRedshiftQueryEditor
-        - arn:aws:iam::aws:policy/AmazonRedshiftQueryEditorV2FullAccess
-      Tags:
-        - Key: sqlworkbench-team
-          Value: codedotorg
-
```

Comment:
This has been removed, in favor of:
* DataAnalytics_FullAccess
* Advocacy_FullAccess

Reminder that "FullAccess" just means maximum access for the job function, as opposed to a companion "*_ReadOnly" role.